app/config/services.yml

```diff
@@ -1,5 +1,5 @@
 parameters:
-    # ...
+    app.notifications.email_sender: 'anonymous@symfony.com'
```

Comment:
What about using example.com as the domain part as it is reserved for such things?